src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,95 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.ui.Ui;
+import duke.task.Deadline;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.ToDo;
+import duke.task.Event;
+import duke.utility.Utility;
+
+import java.time.LocalDateTime;
+
+/**
+ * An AddCommand class that extends from the Command class.
+ *
+ * @author KelvinSoo
+ * @version A-MoreOOP
+ */
+public class AddCommand extends Command{
+
+    private final String command;
+    private final String description;
+
+    /**
+     * A constructor to initialize an add command.
+     * @param command The type of add command eg. todo, deadline, event
+     * @param description the description of the command
+     */
+    public AddCommand(String command, String description) {
+        this.command = command;
+        this.description = description;
+    }
+
+    /**
+     * a method to execute a command.
+     * @param taskList The task list to execute the command on
+     * @param ui The user interface to display the reply
+     * @param storage The place to store the session
+     */

```

Comment:
Need the "@throws" field in Javadocs.

Reply:
I've completely forgotten about it. Thanks for pointing it out for me!